pkg/remote/terraform/provider.go

```diff
@@ -88,8 +92,16 @@ func (p *TerraformProvider) Runner() *parallel.ParallelRunner {
 }
 
 func (p *TerraformProvider) configure(alias string) error {
+	var version string
+	lockFile, diags := LoadLocksFromFile(LockFileName)
```

Comment:
We should log in debug the diags, it will help us investigate some case where we wont be able to read the lockfile for some reason